src/layout/dom.ts

```diff
@@ -217,16 +225,18 @@ function remove(id: number, source: Source): void {
     value.children = [];
 }
 
-function layout(tag: string, id: number, parentId: number): void {
+function meta(tag: string, id: number, parentId: number): void {
```

Comment:
What does "meta" mean? Earlier name seemed clearer to me.

Reply:
Layout was ok when it was all about layout. Now it's more about metadata. I will rename it metadata for now.